config.sh

```diff
@@ -66,3 +66,13 @@ function run_tests {
     # Show BLAS / LAPACK used
     python -c 'import scipy; scipy.show_config()'
 }
+
+function patch_code {
```

Comment:
Actually `config.sh` doesn't get sourced until after this.   I guess it could be sourced before by default.   But - for now - can you put this in it's own file?  As in:

```
./patch_code.sh $REPO_DIR
```

in the travis.yml file?

Reply:
ok, did it like that